source/_lovelace/button.markdown

```diff
@@ -7,10 +7,35 @@ description: "The Button card allows you to add buttons to perform tasks."
 The Button card allows you to add buttons to perform tasks.
 
 <p class='img'>
-<img src='/images/lovelace/lovelace_entity_button_card.png' alt='Screenshot of the button card'>
-Screenshot of the Button card.
+<img src='/images/lovelace/lovelace_entity_button_card.png' alt='Screenshot of the button cards'>
+Screenshot of three Button Cards.
 </p>
 
+## Card Settings
+
+{% configuration_basic %}
+Entity:
+  description: The entity the card interacts with, for example `light.living_room`.
+Name:
+  description: The button name. The default is the name of the entity, if an entity is selected.
+Icon:
+  description: The icon shown on the card. The default is the entity domain icon, if an entity is selected.
+Show Name?:
+  description: A toggle to show or hide the button name.
+Show Icon?:
+  description: A toggle to show or hide the icon.
+Icon Height:
+  description: The height of the icon, in pixels.
+Theme:
+  description: The card theme, which may be set to any installed theme.
```

Comment:
```suggestion
  description: Theme your card using any installed theme.
```

Can you also link `theme` to the them documentation

Reply:
Do we have a style guide other than what’s on the developer documentation standards page? Both the Microsoft style guide and the Google developer documentation style guide suggest writing parameters in the way we have here (starting with ‘the’ or ‘a/an’, with a description that follows). My preference would be to have them all written in the same way.

Reply:
@frenck Is our style guide 😄 but starting with the sounds like a good guide to follow

Reply:
We don't have a style guide on this yet. Both the Microsoft & Google guides are amazing, but we don't enforce it.